mysql/mysql_gtid.go

```diff
@@ -9,8 +9,9 @@ import (
 	"strconv"
 	"strings"
 
+	uuid "github.com/satori/go.uuid"
+
```

Comment:
Seems  don’t need change this

Reply:
it's added by IDE, i haven't figure out why. If i remove it directly, it will fail to compile.